Cookbook.Domain/Models/UnitType.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Cookbook.Domain.Models
+{
+    public enum UnitType
+    {
+        kg = 1,
+        dkg = 2,
+        g = 3,
+
+        l = 4,
+        dl = 5,
+        cl = 6,
+        ml = 7,
+
+        piece = 8,
+        pieces = 9,
+        clove = 10,
+        cloves = 11,
+
+        tablespoon = 12,
+        tablespoons =13, 
```

Comment:
Ezek a plural dolgok nekem nagyon furak. A unit az tablespoon, akar egy, akar ketto. A kifejezes, angolban, nyilvan tablespoons, de magyarban mar pl nem. Mi van ha magyar localization front end fut ugyanezzel a backend-del? Szerintem nem jo amikor egy adott nyelv iranyitja a model-t.

Reply:
hat en singular-t hasznalnek az enum-ban, aztan egy kulon logika hogy ebbol hogy lesz UI string. De ezzel nem is kell igazan foglalkoznod mert nincs front end. A Json-ben is a unit szerintem csak egy "marker" kene legyen, aztan egy logika atkonvertalja majd arra amit a UI designer akar. 
- Magyarban nyilvan mas lesz a szo
- Magyarban nincs is tobbes szam, "2 teaskanalak"
- Lehet telefonon roviditeni akar, a hely miatt "tbsp"
- Lehet recept editorban akar egy dropdown-t ahol kivalaszthatod a unit-ot, ott nem jelenhet meg a singular meg a plural is vagy akkor kene valami elbaszott validator hogy az "1 tablespoons"-t ne fogadja el meg a "2 tablespoon"-t
- Lehet hogy akar valaki majd "scaling" funkcionalitast, hogy beirhasd hogy nem 2 hanem 4 vagy 8 fore akarod megfozni. Akkor az 1 tablespoon-bol ketto lesz, de az nem csak egy szorzassal jarna hanem a unit-ot is meg kene valtoztatnod singular-rol plural-ra
- Lehet hogy be lehet allitani preferred unit-ot, ahol automatikusan konvertalja a front end, akkor megint bonyolultabb ket enum erteket kezelni ugyanarra
- UI-wise maga az enum is inkonzisztens, mert most random lasagne receptekre ranezve, vagy konzisztensen roviditenek (tbsp, gr, ml) vagy konzisztensen kiirjak (tablespoon, grams, milliliter). Ez nem baj ha az enum teljesen decouple-d a UI-tol, azt irsz amit akarsz, de emiatt sem erdemes siman ToString-re bizni a UI string-et.

en pont ugy kezelnem mint a kg, g meg a tobbit. Technically ott is a plural a nyelvtanilag helyes amikor kiirod ("grams", "liters"). Szoval itt szerintem csak enum-ban azt kell jelolnod hogy milyen unit, aztan majd a front end kitalalja hogy azt hogyan kommunikalja le a user-nek.